server/api.js

```diff
@@ -16,4 +16,33 @@ router.get("/clients", (req, res) => {
 	});
 });
 
+router.post("/clients", (req, res) => {
+//querying what the last id in the DB is and creating a new id
+	db.query("SELECT max(client_id) FROM clients;")
+	.then((result) => {
+		const newClientId = result.rows[0].max++;
+		insertNewClient(req, res, newClientId);
```

Comment:
You don't need to create a new client ID since you already have the SERIAL keyword in your clients.client_id column, which will auto increment the ID without any input.
In fact, you are not using the newClientId parameter in insertNewClient, but it works anyway because of SERIAL.
I suggest you remove this first query.

Reply:
Ahhh, I had a feeling it might be unnecessary, could have saved myself some work!

Reply:
Thanks Alessandro, I've removed that now. I also pushed a file I didn't mean to include so that's the other commit :(